commands/audit/audit.go

```diff
@@ -145,11 +150,10 @@ func (auditCmd *AuditCommand) Run() (err error) {
 		}
 	}
 	var messages []string
-	if !auditResults.ExtendedScanResults.EntitledForJas {
+	if !auditResults.EntitledForJas {
 		messages = []string{coreutils.PrintTitle("The ‘jf audit’ command also supports JFrog Advanced Security features, such as 'Contextual Analysis', 'Secret Detection', 'IaC Scan' and ‘SAST’.\nThis feature isn't enabled on your system. Read more - ") + coreutils.PrintLink("https://jfrog.com/xray/")}
 	}
-	if err = utils.NewResultsWriter(auditResults).
-		SetIsMultipleRootProject(auditResults.IsMultipleProject()).
```

Comment:
Why didn't you use SetIsMultipleRootProject in the new ResultsWriter as it was before?

Reply:
we can know if the results have multiple projects with a new method of SecurityCommandResults: `results.HasMultipleTargets()`, so it is used but since we already passing the results no need for the extra setter.
this is only if you want to override and always use specific value